pipeline_tools/input_utils.py

```diff
@@ -218,3 +219,59 @@ def create_optimus_input_tsv(uuid, version, dss_url):
         f.write('{0}'.format(sample_id))
 
     print('Finished writing files')
+
+
+def get_cellranger_inputs(uuid, version, dss_url):
+    """ Get inputs for cellranger count workflow
+
+    Args:
+        uuid (str): the bundle uuid
+        version (str): the bundle version
+        dss_url (str): the DCP Data Storage Service
+
+    Returns:
+        None
+
+    Raises:
+        optimus_utils.LaneMissingFileError if any fastqs are missing
```

Comment:
nit: `optimus_utils.LaneMissingFileError: if any fastqs are missing` (the colon)